partner_communication_switzerland/models/contracts.py

```diff
@@ -483,6 +483,25 @@ def contract_waiting_mandate(self):
 
         return res
 
+    def notify_sds_new_sponsorship(self):
+        self.ensure_one()
+        config_settings = self.env["res.config.settings"].sudo()
+        sds_partner_id = config_settings.get_sponsorship_de_id()
+        if self.partner_id.lang == 'fr_CH':
+            sds_partner_id = config_settings.get_sponsorship_fr_id()
+        if self.partner_id.lang == 'it_IT':
+            sds_partner_id = config_settings.get_sponsorship_it_id()
+        sds_user = self.env['res.users'].sudo().search([('partner_id', '=', int(sds_partner_id))])
+
+        if sds_user.id:
+            self.partner_id.activity_schedule(
```

Comment:
Let's use `correspondent` instead of payer.
```suggestion
        if self.correspondent_id.lang == 'fr_CH':
            sds_partner_id = config_settings.get_sponsorship_fr_id()
        if self.correspondent_id.lang == 'it_IT':
            sds_partner_id = config_settings.get_sponsorship_it_id()
        sds_user = self.env['res.users'].sudo().search([('partner_id', '=', int(sds_partner_id))])

        if sds_user.id:
            self.correspondent_id.activity_schedule(
```